libcaf_core/test/string_view.cpp

```diff
@@ -4,13 +4,15 @@
 
 #define CAF_SUITE string_view
```

Comment:
I think we can just drop the test suite.

Reply:
Sure? I did not delete the implementation of `caf::string_view`, hence the tests are technically still useful..

Reply:
Yeah, I hear you. Since it's "on its way out", I'd be fine with just dropping the tests. However, I don't have much preference either way so feel free to keep it in for now and we remove the tests alongside the implementation later.

Reply:
I'd leave them then.